utils/data.hpp

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include "shapes/Shapes.hpp"
+#include "Canvas.hpp"
+#include "exceptions.hpp"
+#include <QString>
+#include <QChar>
+#include <QMap>
+#include <QColor>
+#include <QFont>
+#include <QFile>
+#include <QFileDialog>
+#include <QTextStream>
+
+namespace jbrush {
+// Directory starts at this string
+const QString STARTING_DIRECTORY = "/";
```

Comment:
There are a lot of constants here, let's give them there own namespace.

Reply:
I put them in a namespace nested inside of "jbrush" and called it "jconstants".  Let me know if that's not what you had in mind!